### ISSUE TITLE
Bioconda package for kent

### ISSUE THREAD

**Issue by jayramr** [ucscGenomeBrowser/kent#70]
Hi Team,

Kindly advise do you have this kent software available as bioconda package? If not do you have any plans to include? Please let me know 

Thanks
Jay

**Comment by galt** [comment]
Yes they are available on bioconda as individual programs rather than a group of utilities. I see someone has separately made a pblat and pblat-cluster in addition to blat, but we are not involved in them.  Please note that commercial use of BLAT requires a license from Kent Informatics.

**Comment by galt** [comment]
all the kent utilities in bioconda are listed as individual packages.

Here is a list of utilities that you can also just download and use for linux
http://hgdownload.soe.ucsc.edu/admin/exe/linux.x86_64/
or mac here
http://hgdownload.soe.ucsc.edu/admin/exe/macOSX.x86_64/

You can use those directly by just downloading them.
You could also try to install with bioconda using the same utility name.

Here is one example I found:
https://anaconda.org/bioconda/blat
which says you can just do this:
conda install -c bioconda blat

We do not maintain the bioconda packages.

**Comment by AngieHinrichs** [comment]
pblat is separate from kent / UCSC.  @galt was pointing out that it exists, but is not the same as blat or UCSC Genome Browser utils despite having a similar name.

Over 250 separate utils from the kent package are already in bioconda.  If there is a util from the kent package that you want to use, there is a good chance that it is already in bioconda.  For example, here are some popular kent utils and their corresponding bioconda packages:

* axtChain: [ucsc-axtchain](https://anaconda.org/bioconda/ucsc-axtchain)
* bedToBigBed: [ucsc-bedtobigbed](https://anaconda.org/bioconda/ucsc-bedtobigbed)
* chainFilter: [ucsc-chainfilter](https://anaconda.org/bioconda/ucsc-chainfilter)

In general, if you want to find a kent util in bioconda, prepend "ucsc-" to the util name, and search for it on https://anaconda.org/bioconda/ .
